test/scripts/jenkins_build_load_testing.sh

```diff
@@ -53,6 +53,9 @@ echo "cloud.auth: ${USER_FROM_VAULT}:${PASS_FROM_VAULT}" >> cfg/metricbeat/metri
 cp cfg/metricbeat/metricbeat.yml $KIBANA_DIR/metricbeat-install/metricbeat.yml
 # Disable system monitoring: enabled for now to have more data
 #mv $KIBANA_DIR/metricbeat-install/modules.d/system.yml $KIBANA_DIR/metricbeat-install/modules.d/system.yml.disabled
+echo " -> Building puppeteer project"
+cd puppeteer
```

Comment:
I see you use `popd` below, should this be `pushd` instead of `cd`?

Reply:
there is a pushd above that moves to the kibana-load-testing folder, which is the sister repo, like the integration-test is. I assume he keeps the puppeteer folder there in order to do the changes faster. So this CD should be fine, at least in my eyes.

Reply:
Ah perfecto mundo :)

Reply:
Marius is right. The idea is to build code here and later run `node build/index.js` via FTR (from Kibana repo). Otherwise I need to build it via FTR as well, that does not sounds good :)